src/components/skills-quiz/data/utils.jsx

```diff
@@ -3,11 +3,38 @@
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
 import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
 
+import { fetchSkillsId, postSkillsGoalsAndJobsUserSelected } from './service';
+import {
+  DROPDOWN_OPTION_CHANGE_CAREERS,
+  DROPDOWN_OPTION_CHANGE_CAREERS_LABEL,
+  DROPDOWN_OPTION_GET_PROMOTED,
+  DROPDOWN_OPTION_GET_PROMOTED_LABEL,
+  DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE,
+  DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE_LABEL,
+  DROPDOWN_OPTION_OTHER,
+  DROPDOWN_OPTION_OTHER_LABEL,
+} from '../constants';
+
 export default function getCommonSkills(content, selectedJobSkills, MAX_VISIBLE_SKILLS) {
   const contentSkills = content.skillNames || [];
   return contentSkills.filter(skill => selectedJobSkills.includes(skill)).slice(0, MAX_VISIBLE_SKILLS);
 }
 
+export const goalLabels = (goal) => {
+  if (goal === DROPDOWN_OPTION_CHANGE_CAREERS) {
+    return DROPDOWN_OPTION_CHANGE_CAREERS_LABEL;
+  }
+  if (goal === DROPDOWN_OPTION_GET_PROMOTED) {
+    return DROPDOWN_OPTION_GET_PROMOTED_LABEL;
+  }
+  if (goal === DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE) {
+    return DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE_LABEL;
+  }
+  if (goal === DROPDOWN_OPTION_OTHER) {
+    return DROPDOWN_OPTION_OTHER_LABEL;
+  }
```

Comment:
Instead of all the if conditions, I think it would be better to have a object of key, value pairs like below

```javascript
dropdown_options = {
    'DROPDOWN_OPTION_CHANGE_CAREERS': DROPDOWN_OPTION_CHANGE_CAREERS_LABEL
}
```

and we can access it like below. 

```javascript
return dropdown_options['DROPDOWN_OPTION_CHANGE_CAREERS']
```

Reply:
As discussed in person it can't be done this way.